common.js

```diff
@@ -0,0 +1,88 @@
+var mzcc = {
```

Comment:
MoZilla Common Code/Crap.
I don't like using things like "common" or "utils" because, well, conflict. so for stuff like this, I tend to load them into a reasonably unique object. Cuts down on accidental conflicts.